e2e/src/main.rs

```diff
@@ -49,44 +49,62 @@ impl Drop for Cleanup {
 
 fn run_with_cleanup() -> Result<()> {
     const CONF_STR: &str = r#"
-  linkup:
-    remote: http://localhost:8787
-  services:
-    - name: frontend
-      remote: https://example.com
-      local: http://localhost:8901
-      rewrites:
-        - source: /foo/(.*)
-          target: /bar/$1
-    - name: backend
-      remote: http://menti.com
-      local: http://localhost:8911
-      # Set dir to this to test env var copies
-      directory: ./
-  domains:
-    - domain: somedomain.com
-      default_service: frontend
-      routes:
-        - path: /api/v1/.*
-          service: backend
-    - domain: api.somedomain.com
-      default_service: backend
-      "#;
+    {
```

Comment:
This should still be yaml - the local configs -> yaml, things the server handles -> JSON

Reply:
Aha! Will take a look into it then!